db/index.js

```diff
@@ -109,3 +110,20 @@ export const migrateStatsDB = async (client) => {
 
   console.log('Migrated `spark-stats` DB schema to version', await postgrator.getDatabaseVersion())
 }
+
+/**
+ * @param {import('./typings.js').Queryable} pgPool
+ * @param {string} day
+ * @param {string[]} participantAddresses
+ */
+export const givenDailyParticipants = async (pgPool, day, participantAddresses) => {
```

Comment:
`db/index.js` is a production code. `getLocalDayAsISOString ` is a test-data-builder helper used in tests; it does not belong to the production code.

I see that `getLocalDayAsISOString` is shared by both `observer/test` and `stats/test`; maybe that's the reason why we decided it's best to move it to spark-evaluate.

Anyhow, let's not overengineer this.

I propose creating a new file, `db/test-helpers.js`, and putting the helper function there.

Reply:
That was my first (actually second, I first attempted to put it in `stats` instead of `db`) attempt, but I was unable to get the other packages to recognize the file if it was not in index.js. See https://github.com/filecoin-station/spark-stats/pull/210/commits/865e0f158ffb75918b60b08bb72fc92a859dd5a8 and https://github.com/filecoin-station/spark-stats/pull/210/commits/81addc2f3559b243144c5c3e222d0047e40afcdb. Any ideas on what needs to be done to make it work?

Reply:
Nice, it worked! When I had originally tried it, I must have done `git add -u .` as I usually do, and forgot that I had a new file that needed to be added. Thoughtless mistake, my bad!